examples/react-native/src/App/App.tsx

```diff
@@ -89,9 +93,11 @@ const WithAuthenticator = React.lazy(
 );
 
 export const ExampleComponent = () => {
-  // .env file or launch argument passed from Detox
-  const APP = EXAMPLE_APP_NAME ?? LaunchArguments.value().EXAMPLE_APP_NAME;
-  switch (APP) {
+  const appName = getExampleAppName();
+
+  console.log(`Running Example App: ${appName}`);
```

Comment:
curious, is the console.log still required, or is this just a nice thing for us to know we're running the right app?

Reply:
Just added it for debugging 😄